src/commands.jl

```diff
@@ -157,3 +158,13 @@ function execute_command(state::DebuggerState, ::Val{:?}, cmd::AbstractString)
     """)
     return false
 end
+
+function execute_command(state::DebuggerState, ::Val{:o}, cmd::AbstractString)
+    frame = active_frame(state)
+    loc = JuliaInterpreter.whereis(frame)
+    loc === nothing && error("could not find source location")
```

Comment:
I don't think this should exit the debugger? Perhaps just `print` and return false?